server/src/modules/data/seed/seed-test.ts

```diff
@@ -1,14 +1,32 @@
 import { config } from "../../../../config/config";
+import { createOrganisation } from "../../actions/organisations.actions";
 import { createUser } from "../../actions/users.actions";
 
 export const seedTest = async () => {
+  const organisationId = await createOrganisation({
+    nom: "Beta gouv",
+    email_domains: [
+      "beta.gouv.fr",
+      "recette.bal.apprentissage.beta.gouv.fr",
+      "preview.bal.apprentissage.beta.gouv.fr",
+      "test.fr",
+    ],
+    etablissements: {
+      nom: "Dinum",
+      siret: "13002526500013", // Siret Dinum
+      is_hq: true,
+      is_close: false,
+    },
```

Comment:
ce serait pas un array ici ?
```suggestion
    etablissements: [{
      nom: "Dinum",
      siret: "13002526500013", // Siret Dinum
      is_hq: true,
      is_close: false,
    }],
```